src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml

```diff
@@ -5,7 +5,7 @@
     var element = (ReCaptcha)Model.Element;
     var publicKey = (string)Model.PublicKey;
 }
-@if (!String.IsNullOrWhiteSpace(publicKey)) {
+@if (HasText(publicKey)) {
```

Comment:
We talked about it during the triage, and I think I commented on the "request" from Nick. @sfmskywalker  is right, you can just use `String.IsNullOrEmptyWhitespace` here. But `HasText` begin technically correct we didn't bother accepting the PR.